### PR TITLE
Adding FPAset and queries from @sarahhunt

### PR DESCRIPTION
**Pull request by david4096**
This PR addresses the need to serve multiple g2psets from a single server by plucking changes @sarahhunt proposed here: https://github.com/ohsu-computational-biology/schemas/pull/1/files.

Also adding toctree and moving G2P docs to docs/api. @bwalsh @dcolligan

### CODE REVIEW

File: src/main/resources/avro/genotypephenotype.avdl [hunk 1]
```diff
@@ -131,6 +151,12 @@ such as resulting from alternative experiments and analysis.
 */
 record FeaturePhenotypeAssociation {
   string id;
+  
+  /** 
+  The ID of the PhenotypeAssociationSet this FeaturePhenotypeAssociation
+  belongs to. 
+  */
+  string phenotypeAssociationId;
```
Comment (jeromekelleher) [inline src/main/resources/avro/genotypephenotype.avdl]: Typo here, should be phenotypeAssociationSetId

**Comment by jeromekelleher** [comment]
We definitely need something like this. Is everyone happy with the PhenotypeAssociationSet terminology?

File: src/main/resources/avro/genotypephenotype.avdl (hunk 1 above)
Comment (david4096) [inline src/main/resources/avro/genotypephenotype.avdl]: Thanks nice catch @jeromekelleher , updated :)

**Comment by david4096** [comment]
The other comment that I'll make here (thanks @diekhans) is that the list of `Feature` in a FPA should probably be a list of `featureId`. A query against sequence annotations with that featureId will give the entirety of the record.

File: src/main/resources/avro/genotypephenotypemethods.avdl [hunk 2]
```diff
@@ -101,6 +150,11 @@ full feature/phenotype/evidence objects.
 */
 record SearchGenotypePhenotypeRequest {
 
+  /**
+  The `PhenotypeAssociationSet` to search.
+  */
+  string phenotypeAssociationSetId;
+
   union {null, string, ExternalIdentifierQuery, OntologyTermQuery,
```
Comment (david4096) [inline src/main/resources/avro/genotypephenotypemethods.avdl]: These need to be better specified and documented. For example, as @diekhans mentioned, does it make sense to have an external  ID query on `Evidence` when evidence doesn't have an external ID field? What are the strings matching on?

**Comment by bwalsh** [comment]
+1 looks good

**Pull request merged**